roles/openshift_logging/README.md

```diff
@@ -69,6 +69,9 @@ When `openshift_logging_install_logging` is set to `False` the `openshift_loggin
 - `openshift_logging_fluentd_buffer_size_limit`: Buffer chunk limit for Fluentd. Defaults to 1m.
 - `openshift_logging_fluentd_file_buffer_limit`: Fluentd will set the value to the file buffer limit.  Defaults to '1Gi' per destination.
 
+- `openshift_logging_audit_container_engine`: When `openshift_logging_audit_container_engine` is set to `True`, the audit log of the container engine will be collected and stored in ES.
+- `openshift_logging_audit_file`: By default the following location of the log file is assumend: `/var/log/audit/audit.log`
+- `openshift_logging_audit_pos_file`: By default the following location of the position file is assumend: `/var/log/audit/audit.log.pos`
```

Comment:
"Location of fluentd in_tail position file for the audit log file.  The default is `/var/log/audit/audit.log.pos`"